ui/components/app/transaction-total-banner/index.scss

```diff
@@ -0,0 +1,20 @@
+.transaction-total-banner {
+  text-align: center;
+
+  &__total {
+    font-size: 40px;
```

Comment:
The line-heights and font-sizes are determined by our typography system, this should be @include H1 (2.5REM which is 2.5 * 16 (default) = 40) and line-heights are set as a percentage. 

this could also be replaced by:
```
<Typography variant={TYOGRAPHY.H1} tag="span" color={COLORS.BLACK} />
```